packages/core/test/zodiac/OptimisticGovernor.js

```diff
@@ -1,691 +1,601 @@
+    identifierWhitelist = await IdentifierWhitelist.deployed();
+    optimisticOracle = await OptimisticOracle.deployed();
+    testToken = await TestnetERC20.new("Test", "TEST", 18).send({ from: accounts[0] });
+    testToken2 = await TestnetERC20.new("Test2", "TEST2", 18).send({ from: accounts[0] });
+
+    // Deploy new MockOracle so that OptimisticOracle disputes can make price requests to it:
+    mockOracle = await MockOracle.new(finder.options.address, timer.options.address).send({ from: owner });
+    await finder.methods
+      .changeImplementationAddress(utf8ToHex(interfaceName.Oracle), mockOracle.options.address)
+      .send({ from: owner });
+    await identifierWhitelist.methods.addSupportedIdentifier(identifier).send({ from: owner });
+  });
+
+  beforeEach(async function () {
+    // Deploy new contracts with clean state and perform setup:
+    avatar = await TestAvatar.new().send({ from: owner });
+    bondToken = await ERC20.new("BOND", "BOND", 18).send({ from: owner });
+    await bondToken.methods.addMember(TokenRolesEnum.MINTER, owner).send({ from: owner });
+    await collateralWhitelist.methods.addToWhitelist(bondToken.options.address).send({ from: owner });
+    await store.methods.setFinalFee(bondToken.options.address, { rawValue: finalFee }).send({ from: owner });
+
+    optimisticOracleModule = await OptimisticGovernor.new(
+      finder.options.address,
+      avatar.options.address,
+      bondToken.options.address,
+      bond,
+      rules,
+      identifier,
+      liveness,
+      timer.options.address
+    ).send({ from: owner });
+
+    avatar.methods.setModule(optimisticOracleModule.options.address).send({ from: owner });
+
+    await bondToken.methods.mint(proposer, doubleTotalBond).send({ from: owner });
+    await bondToken.methods.approve(optimisticOracleModule.options.address, doubleTotalBond).send({ from: proposer });
+    await bondToken.methods.mint(disputer, totalBond).send({ from: owner });
+    await bondToken.methods.approve(optimisticOracle.options.address, totalBond).send({ from: disputer });
+  });
+
+  it("Constructor validation", async function () {
+    // 0 liveness.
+    assert(
+      await didContractThrow(
+        OptimisticGovernor.new(
+          finder.options.address,
+          avatar.options.address,
+          bondToken.options.address,
+          bond,
+          rules,
+          identifier,
+          0,
+          timer.options.address
+        ).send({ from: owner })
+      )
+    );
+
+    // Unapproved token.
+    assert(
+      await didContractThrow(
+        OptimisticGovernor.new(
+          finder.options.address,
+          avatar.options.address,
+          (await ERC20.new("BOND", "BOND", 18).send({ from: owner })).options.address,
+          bond,
+          rules,
+          identifier,
+          liveness,
+          timer.options.address
+        ).send({ from: owner })
+      )
+    );
+
+    // Unapproved identifier.
+    assert(
```

Comment:
You should test that `setUp` can only be called once and performs expected behavior